packages/storage-web-api/src/controllers/submit-website-scan-controller.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as ApiContracts from 'api-contracts';
+import * as StorageDocuments from 'storage-documents';
+import { RestApiConfig, ServiceConfiguration } from 'common';
+import { inject, injectable } from 'inversify';
+import _ from 'lodash';
+import { ContextAwareLogger } from 'logger';
+import { ApiController, HttpResponse, WebApiErrorCodes, WebsiteProvider, WebsiteScanProvider } from 'service-library';
+import { createWebsiteScanApiResponse, WebsiteScanDocumentResponseConverter } from '../converters/website-scan-document-response-converter';
+import { SubmitWebsiteScanRequestValidator } from '../request-validators/submit-website-scan-request-validator';
+
+const defaultFrequencyKeys: { [key in StorageDocuments.ScanType]: keyof RestApiConfig } = {
+    a11y: 'defaultA11yScanFrequency',
+    privacy: undefined,
+    security: undefined,
+};
+
+@injectable()
+export class SubmitWebsiteScanController extends ApiController {
```

Comment:
We may want to follow HTTP verbs pattern in controllers naming like GET/POST/PUT/DELETE ie `SubmitWebsiteScanController` will be `PostWebsiteScanController`

Same for `class UpdatePageController`